docs/templates/templates.md

```diff
@@ -1,5 +1,28 @@
-Most dynamic text such as join messages, custom commands and such is generated using templates (more specifically [GO's text/template package](https://golang.org/pkg/text/template/))
+The go template engine is used for YAGPDB's custom commands.
+This page aims to help you to get the most out of the custom commands feature.
 
-This page aims to explain how to get the most out of them, and what functions are provided
+Available template data:
 
-- TODO -
\ No newline at end of file
+> type User
+* `{{.User.Username}}` Outputs the username
```

Comment:
remove the {{ and }}'s as the template engine also runs on this doc page, and will try to parse {{.User.Username}} which will fail here, just make a note to surround shit in {{...}}.